docs/examples/fork-process-model/flask-gunicorn/README.rst

```diff
@@ -8,4 +8,4 @@ Run application
 ---------------
 .. code-block:: sh
 
-    gunicorn app -c gunicorn.config.py
```

Comment:
Explicit is better than implicit. I would still keep it.

Reply:
Fair point. Updated the README.